DIMS-Core.Database/DIMS-Core.Database/Views/vUserTrack.sql

```diff
@@ -0,0 +1,7 @@
+create or alter view vUserTrack 
+as
+select UP.UserId, T.TaskId, TT.TaskTrackId, T.Name as TaskName, TT.TrackNote, TT.TrackDate
```

Comment:
You can user UserId column from UserTask table so you don't have to join UserProfiles table

Reply:
Fixed